### ISSUE TITLE
Add convenient keyboard-only input support

### ISSUE THREAD

**Issue by AlexYegupov** [allcount/allcountjs#50]
Currentry it seems that admin interface requires active mouse usage.

Imagine the situation that user want to input some 50 items in admin interface. It's almost impossible without keyboard only  especially for pressing "Save" and "Create" buttons. 

Maybe it could be useful for many of potential admin users to optimize  tab order of page controls and make other tunes for keyboard-only convenient data input.

**Comment by paveltiunov** [comment]
Hi @AlexYegupov !
Thanks for posting this issue!
Seems pretty interesting. Do you have any proposal of how it could be done?
Do you propose to have keyboard shortcuts for "Save" and "Create" operations?
I believe by default tab order is an order of fields in entity definition. What optimizations you propose to do here? Something like defining `tabindex`?

**Comment by AlexYegupov** [comment]
Yes, the simplest solution probably would be enough in 80% of cases:
 - set tabindex too all input controls on page. For rarely-used controls set tabindex to "-1". 
For fields keep declaration order. After last control most probably the "Save" button should be focused. Then - "Cancel". 
 - set autofocus html attribute for most-probable control on the form

Also possible options:
 - Add titles to all buttons (maybe as disablealble option) to make user possible to activate desired button by finding it's caption as text on page (ctrl+f) and pressing escape. In that case any button 
 - Imitate or implement classical html form behavour with using "default submit button". That means pressing "Enter" on any text input control == pressing form submit button (that in case of edit form probably means "save" button).


Any solution could be tested very easily: creating multiple items (and subform items) in admin forms should be performed easily by using Tab and Enter only keys for saving/swithing/cancelling operations. (like it is on good-organized  desktop applications).
